### PR TITLE
Lps 159687: There are two IDs on Title Field

### PR DESCRIPTION
**Pull request by thienquach87**
@quanhuynhces 
Here is the PR for fixing https://issues.liferay.com/browse/LPS-159687. Please take time to help me review it.
Thanks!
Thien

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp [hunk 1]
```diff
@@ -98,15 +101,22 @@ renderResponse.setTitle(LanguageUtil.format(request, "edit-x", objectDefinition.
...
 					>
 						<aui:select disabled="<%= !objectDefinitionsDetailsDisplayContext.hasUpdateObjectDefinitionPermission() %>" name="titleObjectFieldId" showEmptyOption="<%= false %>">
-							<aui:option label='<%= LanguageUtil.get(request, "id") %>' selected="<%= true %>" value="" />
+							<aui:option label="<%= HtmlUtil.escape(objectIdField.getLabel(locale)) %>" selected="<%= true %>" value="<%= objectIdField.getObjectFieldId() %>" />
 
 							<%
 							for (ObjectField objectField : nonrelationshipObjectFields) {
+								String idFieldLabel = HtmlUtil.escape(
+									objectIdField.getLabel(locale));
+								String objectFieldLabel = HtmlUtil.escape(objectField.getLabel(locale))
```
Comment (quanhuynhces) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp:110]: You forget the semicolon in this line.

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java [hunk 2]
```diff
@@ -162,6 +162,20 @@ public ObjectDefinition getObjectDefinition() {
 			ObjectWebKeys.OBJECT_DEFINITION);
 	}
 
+	public ObjectField getObjectFieldbyName(
```
Comment (quanhuynhces) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java:165]: I think we should refactor this method name to getObjectFieldByName to follow the camel-case naming convention.

**Review by quanhuynhces — COMMENTED** [review]
image

I think you should format source to this for cleaner code.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp (hunk 1 above)
Comment (thienquach87) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp]: Thank you, I updated code

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java (hunk 2 above)
Comment (thienquach87) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java]: Thank you, I updated code

**Comment by quanhuynhces** [comment]
forward https://github.com/liferay-objects/liferay-portal/pull/893

**Pull request closed without merging**